tests/manage/z_cluster/test_restart_mgr_while_two_mons_down.py

```diff
@@ -0,0 +1,111 @@
+import logging
+import pytest
+import time
+
+from ocs_ci.ocs.utils import get_pod_name_by_pattern
+from ocs_ci.ocs import constants, ocp, defaults
+from ocs_ci.framework import config
+from ocs_ci.ocs.resources.pod import (
+    get_mgr_pods,
+    check_pods_in_running_state,
+    wait_for_pods_to_be_running,
+)
+from ocs_ci.utility.utils import TimeoutSampler
+from ocs_ci.ocs.resources.pod import get_deployments_having_label
+from ocs_ci.framework.testlib import (
+    ManageTest,
+    tier2,
+    bugzilla,
+    polarion_id,
+)
+
+log = logging.getLogger(__name__)
+
+
+@tier2
+@bugzilla("1990031")
+@polarion_id("OCS-2696")
+class TestRestartMgrWhileTwoMonsDown(ManageTest):
+    """
+    Restart mgr pod while two mon pods are down
+
+    """
+
+    @pytest.fixture(scope="function", autouse=True)
+    def teardown(self, request):
+        """
+        Verify all pods on openshift-storage project on Running state
+
+        """
+
+        def finalizer():
+            for mon_scale in self.mons_scale:
+                self.oc.exec_oc_cmd(f"scale --replicas=1 deployment/{mon_scale}")
+            wait_for_pods_to_be_running(timeout=600)
+
+        request.addfinalizer(finalizer)
+
+    def test_restart_mgr_while_two_mons_down(self):
+        """
+        Test Procedure:
+        1.Scaling down two mons:
+        oc scale --replicas=0 deploy/rook-ceph-mon-a
+        oc scale --replicas=0 deploy/rook-ceph-mon-b
+
+        2.Restarting mgr
+        oc delete pod -l app=rook-ceph-mgr
+
+        3.sleep 5 seconds
+
+        4.Scaling mons back up
+        oc scale --replicas=1 deploy/rook-ceph-mon-a
+        oc scale --replicas=1 deploy/rook-ceph-mon-b
+
+        5.sleep 10
+
+        6.Waiting for mgr pod move to running state:
+        oc get pod -l app=rook-ceph-mgr
+
+        """
+        self.oc = ocp.OCP(
+            kind=constants.DEPLOYMENT, namespace=config.ENV_DATA["cluster_namespace"]
+        )
+        mons = [
+            mon["metadata"]["name"]
+            for mon in get_deployments_having_label(
+                constants.MON_APP_LABEL, defaults.ROOK_CLUSTER_NAMESPACE
+            )
```

Comment:
You can use the method `get_mon_pods` here instead of `get_deployments_having_label(
                constants.MON_APP_LABEL, defaults.ROOK_CLUSTER_NAMESPACE
            )`

Reply:
I need to find the label (no pod names)
![image](https://user-images.githubusercontent.com/61982127/144825698-f658dcdf-e910-4c20-bff8-1b1448307950.png)

Reply:
Yes, I see, here you need to find the mon deployments and not the mon pods.